internal/lookout/repository/schema/statik/statik.go

```diff
@@ -9,6 +9,6 @@ import (
 const LookoutSql = "lookout/sql" // static asset namespace
 
 func init() {
```

Comment:
From this file and below, do you know why these were included?  Looks like we committed code unrelated to pulsar upgrade as I hope lookout is unrelated to Pulsar.

Reply:
I agree - Lookout is unrelated to Pulsar (as far as I know). When I did the Pulsar 3 upgrade, I had to upgrade the `pulsar-client-go` library (see the (very few) changes in `go.mod`), and they changed the signature on about 3 functions to start returning an `error` (when they returned nothing before). So I fixed our interface definitions that needed those changes, which required regenerating the mocks - that last step caused all the changes in the lookout/lookoutv2 dirs, including that giant statik definition. I thought about seeing if I could revert some of the files that were changed by `generate`, but compilation and testing quickly failed, so I just stayed with clean generated updates (I did no manual tweaks to those files after running the generate step).